docs/docs/middlewares/url.md

```diff
@@ -21,7 +21,8 @@ url:
 ```
 
 ## Provides
-name | type     | description
------|----------|------------
-url  | string   | The canonical URL. 
+name    | type     | description
+--------|----------|------------
+url     | string   | The canonical URL with parent domain (if preset in `parameters.yml`)
```

Comment:
Please change description to more precise wording:

The URL where the site runs on current environment. It consists of the canonical URL as defined in `site.yml` and `parent_domain` as defined in `parameters.yml`.